lib/utils/log/slog_handler.go

```diff
@@ -44,6 +44,16 @@ const TraceLevel = slog.LevelDebug - 1
 // TraceLevelText is the text representation of Trace verbosity.
 const TraceLevelText = "TRACE"
 
+// DiscardHandler is a [slog.Handler] that discards all messages. It
```

Comment:
Might want to add a TODO here to remove this when we upgrade to Go 1.24 since the stdlib will ship with a discard handler: https://pkg.go.dev/log/slog@master#DiscardHandler